client/css/style.css

```diff
@@ -2216,22 +2211,11 @@ part/quit messages where we don't load previews (adds a blank line otherwise) */
 
 @media (min-width: 480px) {
 	/* Fade out for long usernames */
-
 	#chat .from {
 		padding-left: 10px;
+		-webkit-mask-image: linear-gradient(to left, transparent, black 10px);
+		mask-image: linear-gradient(to left, transparent, black 10px);
```

Comment:
Mind making this `20px` before merging?

Reply:
As you noted in your other PR, increasing the distance here starts fading nicks that are not overflowing. I set it at 10px because that's the padding.